rcl/test/rcl/test_graph.cpp

```diff
@@ -118,6 +123,10 @@ class CLASSNAME (TestGraphFixture, RMW_IMPLEMENTATION) : public ::testing::Test
     delete this->wait_set_ptr;
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
 
+    if (rcl_logging_rosout_enabled()) {
```

Comment:
Since we keep this code in test_graph.cpp, I think this check condition (and the related PRs) should be addressed similarly to test_logging_rosout.cpp I mentioned before.

Reply:
https://github.com/ros2/rcl/pull/1065/commits/d29753fb01ce5d1c1264fce9371f27aee5694dd1